server/app.js

```diff
@@ -17,16 +18,27 @@ app.use(urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use(express.static(join(__dirname, "public")));
 
+// MongoDB config
+const mongoUri = process.env.MONGO_DB_ATLAS_URI;
+mongoose
+  .connect(mongoUri, {
+    useNewUrlParser: true,
+    useCreateIndex: true,
+    useUnifiedTopology: true,
+  })
+  .then(() => console.log("MongoDB successfully connected"))
+  .catch((error) => console.error(error));
+
```

Comment:
Looks good, I'd recommend this be put a different file that is imported here, just to keep app.js lean